src/bundler.js

```diff
@@ -248,10 +250,12 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
     var allJsArr = [], allMinJsArr = [], index = 0, pending = 0;
     var whenDone = function () {
 
+        var sourceMap = options.sourcemaps && !options.webpack;

```

Comment:
Try to make Bundler place nicely with the source maps generated by webpack in Capi projects is a bit thorny. For simplicity, I've just disabled source maps for webpack bundles.